src/edu/ucsb/nceas/metacat/systemmetadata/SystemMetadataManager.java

```diff
@@ -226,53 +237,73 @@ public void store(SystemMetadata sysmeta, boolean changeModifyTime, DBConnection
+                                                + "the system metadata for pid " + pid.getValue()
+                                                + " since " + e.getMessage());
+                } catch (InvalidSystemMetadata e) {
+                    throw new InvalidRequest("0000", "SystemMetadataManager.store - can't store "
+                                            + "the system metadata for pid " + pid.getValue()
+                                            + " since " + e.getMessage());
+                } catch (AccessException e) {
+
+                    throw new InvalidRequest("0000", "SystemMetadataManager.store - can't store "
+                                                + " the system metadata for pid " + pid.getValue()
+                                                + " since " + e.getMessage());
+                } catch (RuntimeException e) {
+                    throw new ServiceFailure("0000", "SystemMetadataManager.store - can't store "
+                                                + "the system metadata for pid " + pid.getValue()
+                                                + " since " + e.getMessage());
+                } catch (IOException e) {
+                    throw new ServiceFailure("0000", "SystemMetadataManager.store - can't store "
+                                                + "the system metadata for pid " + pid.getValue()
+                                                + " since IOException " + e.getMessage());
+                } catch (MarshallingException e) {
+                    throw new ServiceFailure("0000", "SystemMetadataManager.store - can't store "
+                            + "the system metadata for pid " + pid.getValue()
+                            + " since a MarshallingException " + e.getMessage());
+                } catch (NoSuchAlgorithmException e) {
+                    throw new ServiceFailure("0000", "SystemMetadataManager.store - can't store "
+                            + "the system metadata for pid " + pid.getValue()
+                            + " since " + e.getMessage());
+                } catch (InterruptedException e) {
+                    throw new ServiceFailure("0000", "SystemMetadataManager.store - can't store "
+                            + "the system metadata for pid " + pid.getValue()
+                            + " since " + e.getMessage());
+                } catch (InvocationTargetException e) {
+                    throw new ServiceFailure("0000", "SystemMetadataManager.store - can't store "
+                        + "the system metadata for pid " + pid.getValue()
+                        + " since " + e.getMessage());
+                } catch (IllegalAccessException e) {
+                    throw new ServiceFailure("0000", "SystemMetadataManager.store - can't store "
+                        + "the system metadata for pid " + pid.getValue()
+                        + " since " + e.getMessage());
```

Comment:
since all these perform almost exactly the same action, can we condense them all into one single `catch`, like:

```java
 } catch (McdbDocNotFoundException | SQLException | InvalidSystemMetadata // [...etc]
```
```